ctapipe/core/traits.py

```diff
@@ -138,22 +139,51 @@ def has_traits(cls, ignore=("config", "parent")):
     return bool(set(cls.class_trait_names()) - set(ignore))
 
 
+class TelescopePatternList(UserList):
+    """
+    Representation for a list of telescope pattern tuples. This is a helper class
+    used  by the Trait TelescopeParameter as its value type
+    """
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        self._lookup = None
+        self._old_getitem = self.__getitem__
```

Comment:
What is this for? `__getitem__` is not overriden.

Reply:
forgot to remove that when I did the last change - I'll remove it (the old implementation overrode `__getitem__`)